judge/models/problem_data.py

```diff
@@ -91,4 +92,4 @@ class ProblemTestCase(models.Model):
     output_limit = models.IntegerField(verbose_name=_('output limit length'), blank=True, null=True)
     checker = models.CharField(max_length=10, verbose_name=_('checker'), choices=CHECKERS, blank=True)
     checker_args = models.TextField(verbose_name=_('checker arguments'), blank=True,
-                                    help_text=_('Checker arguments as a JSON object.'))
+                                    help_text=_('Checker arguments as a JSON object.'))
```

Comment:
Can you add the trailing `\n` so that git doesn't complain?

Reply:
Sorry, do you mean at the end of the file?

Reply:
Yes.

Reply:
Done!